### DISCUSSION TITLE
Google Tag Manager (GTM) integration

### DISCUSSION THREAD

**Discussion by NEDIMA951** [bullhorn/career-portal#465] [Q&A] [answered]
Please help, how to add Google Tag Manager (GTM) integration on Bullhorn Career Portal WordPress Plugin pages. I find how to add Analytics, but I need only Tag Manager. I try to find any documentation integration with Bullhorn , but found nothing.

**Comment by charlesabarnes** [comment, marked as answer]
Tag manager is not something explicitly supported so there are no instructions.  

You can install the HTML snippet in the index.html file of the wordpress plugin.

**Reply by NEDIMA951** [reply]
Thank you very much! I thought that I need to edit app.json. I have tried to do by analogy with Google Analytics integration. That's why I missed this opportunity. You the best. Best wishes!
